vtm-extras/src/org/oscim/tiling/source/overpass/OverpassTileSource.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2013 Hannes Janetzek
+ * Copyright 2018 devemux86
+ * Copyright 2019 Gustl22
+ *
+ * This file is part of the OpenScienceMap project (http://www.opensciencemap.org).
+ *
+ * This program is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.oscim.tiling.source.overpass;
+
+import org.oscim.core.BoundingBox;
+import org.oscim.core.Tile;
+import org.oscim.tiling.ITileDataSource;
+import org.oscim.tiling.OverzoomTileDataSource;
+import org.oscim.tiling.source.UrlTileDataSource;
+import org.oscim.tiling.source.UrlTileSource;
+import org.oscim.utils.overpass.OverpassAPIReader;
+
+public class OverpassTileSource extends UrlTileSource {
+
+    private static final String DEFAULT_URL = "https://www.overpass-api.de/api/interpreter?data=[out:json];";
+    private static final String DEFAULT_PATH = "(node{{bbox}};way{{bbox}};>;);out%20body;";
```

Comment:
So this won't fetch relations? Does this mean there are no boundaries and potentially no shorelines?

Reply:
I have not yet tested relations. They have to be managed (e.g. like in mapsforge writer) and I wanted to provide a quick and simple overpass source. So some elements might not be classified correctly. (E.g. in my case boundaries and coastlines were drawn, but they also can vanish).
But anyone is welcomed to extend preprocessing.
As mentioned in #663 would be good to add custom filters/queries in 'Overpass QL'.

Reply:
Ok thanks, I was just wondering :)